Changelog.md

```diff
@@ -12,6 +12,10 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
 - [#660](https://github.com/timescale/timescaledb-toolkit/issues/660): Heartbeat aggregate rollup should interpolate aggregates
 - [#679](https://github.com/timescale/timescaledb-toolkit/issues/679): Heartbeat agg rollup producing invalid aggregates.
 
+#### Stabilized features
+
+- [#650](https://github.com/timescale/timescaledb-toolkit/pull/650): Stabilize interpolated_delta, interpolated_rate, and interpolated_average.
```

Comment:
- [#650](https://github.com/timescale/timescaledb-toolkit/pull/650): Stabilize interpolated_delta & interpolated_rate for counter aggregate, and interpolated_average for time-weighted aggregate.